client/my-sites/plans/jetpack-plans/product-grid/products-order.ts

```diff
@@ -17,11 +18,15 @@ import {
 	PRODUCT_JETPACK_BACKUP_REALTIME_MONTHLY,
 } from '@automattic/calypso-products';
 
+/**
+ * Internal dependencies
+ */
+import { getForCurrentCROIteration, Iterations } from '../iterations';
+
 /**
  * Type dependencies
  */
 import type { JetpackProductSlug } from 'calypso/lib/products-values/types';
-import type { JetpackPlanSlugs } from '@automattic/calypso-products';
```

Comment:
Did this raise a TS error?

Reply:
Since it's still there as an import on line 5, TypeScript's happy 🙂

Reply:
Sorry, that was unclear. I was wondering why you moved `JetpackPlanSlugs`, but now that I see it again it seems that maybe it was to have it under the existing `calypso-products` import. I originally thought it could be because you spotted a TS error and were trying to fix.

Reply:
Yup, it was to have all the calypso-products imports in one expression. I flip-flop as to which form I like better though, so if you or anyone else has a preference or knows an agreed-upon best practice for imports like this, I'll adopt it going forward. 👍